proto/viam/service/motion/v1/motion.proto

```diff
@@ -33,20 +33,16 @@ service MotionService {
     };
   }
 
+  // Generate and begin executing an execution to move a component
+  // to a specific GPS coordinate.
+  // May replan to avoid obstacles & account for location drift.
+  // Creates a new plan upon replanning.
```

Comment:
I think this is a little confusing without a greater description of what these terms mean, which IMO is what our docs page is for.  Maybe we should just keep it general rather than implying things about what happens under the hood?

Reply:
Actually none of the other methods even have comments on them so I think its maybe fine to not add this?  I would hope that people would refer to our docs page for usage, etc.

Reply:
discussed offline, I believe these comments are used by some SDKs